packages/collector/src/tracing/instrumentation/process/childProcess.js

```diff
@@ -24,6 +32,7 @@ logger = require('../../../logger').getLogger('tracing/child_process', newLogger
 // There is also ./edgemicro.js, which is responsible for instrumenting the code that is used to spawn the individual
 // edgemicro workers.
 exports.init = function() {
+  cls = getCls();
```

Comment:
It  might be better to call `getCls` where `cls`  is actually used (around line 103), instead of calling it in init.

We also nee to add a null check there, because `getCls` might not return a value in some cases.

See `packages/collector/src/uncaught/index.js` for an example.